proto/cdcpb.proto

```diff
@@ -75,12 +75,16 @@ message Event {
         uint64 commit_ts = 2;
         LogType type = 3;
 
+        // generation is for pipelined DML protocol. See kvrpcpb.FlushRequest.generation.
+        uint64 generation = 10;
+
```

Comment:
should we put it after the 9th field?

Reply:
Not necessary. If other guys add more fileds with same id, proto compiler will report it. So we can always group fields by meanning.